src/events/messageCreate.ts

```diff
@@ -1,15 +1,14 @@
-import { Logger } from 'winston';
+import axios from 'axios';
 import { Client, Message } from 'discord.js';
+import { readFileSync } from 'fs';
+import { writeFile } from 'fs/promises';
+import { PDFDocument } from 'pdf-lib';
+import { Logger } from 'winston';
 import { applyBonusByUserId } from '../components/coin';
 import { vars } from '../config';
 import { sendKickEmbed } from '../utils/embeds';
 import { convertPdfToPic } from '../utils/pdfToPic';
 
-import { readFileSync } from 'fs';
-import { writeFile } from 'fs/promises';
-import axios from 'axios';
-import { PDFDocument } from 'pdf-lib';
-
 const ANNOUNCEMENTS_CHANNEL_ID: string = vars.ANNOUNCEMENTS_CHANNEL_ID;
 const RESUME_CHANNEL_ID: string = vars.RESUME_CHANNEL_ID;
 const IRC_USER_ID: string = vars.IRC_USER_ID;
```

Comment:
What do you think about moving all consts (across all files) to files in a constants directory or something similar? (as I mentioned in an earlier comment)